services/movie.py

```diff
@@ -15,13 +17,17 @@ def get_movies(
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
+@atomic
```

Comment:
probably better use transtaction.atomic() but not sure. Mb mentors can help with this)

Reply:
yeah, it is more usual to use @transaction.atomic decorator,but it think this is not critical